webapp/channels/package.json

```diff
@@ -161,7 +161,6 @@
     "jest-cli": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "jest-junit": "16.0.0",
-    "jest-styled-components": "7.2.0",
```

Comment:
why did we had that 🤔

Reply:
I'm not sure, but I assume we needed it at some point